Gopkg.lock

```diff
@@ -2,18 +2,23 @@
 
 
 [[projects]]
+  digest = "1:8e47871087b94913898333f37af26732faaab30cdb41571136cf7aec9921dae7"
```

Comment:
Were the updates to this file generated by `make vendor`? Looks like the you must have a newer version of `go dep`

Reply:
I'm currently running v0.5.0. Should I back rev to an earlier version?